cfd_protocol/tests/cfds.rs

```diff
@@ -372,6 +372,443 @@ fn create_cfd() {
         .expect("valid punish transaction signed by taker");
 }
 
+#[test]
+fn renew_cfd() {
```

Comment:
I guess this asks for some refactoring and code-reuse between the two different tests :)